batch7crud-roll6/src/main/java/com/lftechnology/batch7crud/db/DbConnector.java

```diff
@@ -0,0 +1,36 @@
+package com.lftechnology.batch7crud.db;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import javax.naming.Context;
+import javax.naming.InitialContext;
+import javax.naming.NamingException;
+import javax.sql.DataSource;
+
+public class DbConnector {
+
+    private static final Logger logger = Logger.getLogger("UserController");
+
+    private DbConnector() {
+
+    }
+
+    public static Connection getMySqlConnection() throws SQLException {
+        Connection connection = null;
+        try {
+            Context initCtx = new InitialContext();
+
+            Context envCtx = (Context) initCtx.lookup("java:comp/env");
+
+            DataSource ds = (DataSource) envCtx.lookup("jdbc/dbInitial");
+            connection = ds.getConnection();
+        } catch (NamingException e) {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)